sortinghat/core/db.py

```diff
@@ -290,6 +291,77 @@ def delete_organization(trxl, organization):
                        target=op_args['organization'])
 
 
+def add_group(trxl, organization, parent, group_name):
```

Comment:
As `organization` and `parent` can be `None`, I think it's better to reformulate the definition of the function this way:
```
def add_group(trxl, group_name, organization=None, parent=None):
```

Reply:
Done 👍